README.md

```diff
@@ -62,6 +62,24 @@
     </td>
   </tr>
 
+  <tr><td><h4>Changelog</h4></td></tr>
+  <tr>
+    <td>
+      <strong>0.2.1</strong><br/>
+      - Removes the LZString compression for backups (page revisions) as per the conversation [here](https://github.com/infinitered/thesis-phoenix/issues/129). <br/>
+      - <em>Adds migration (run `mix thesis.install && mix ecto.migrate`)</em>. It is important to do this before 0.3.0 as the LZString dependency will be removed then.<br/>
+      - Adds [notifications](https://github.com/infinitered/thesis-phoenix/issues/147). This allows us to warn users of potential Thesis issues depending on page state, environment, Thesis version, etc. Notifications can also be set in the host app config to include custom logic. <br/>
+      - Bug fixes: [page revision exception](https://github.com/infinitered/thesis-phoenix/issues/128), [add new page button hover issue](https://github.com/infinitered/thesis-phoenix/issues/148).
+      <br/><br/>
+      <strong>0.2.0</strong><br/>
+      - Adds a backup (page revision history) feature. Every time a page is saved, a snapshot of the page and content is captured and can be restored later. <br/>
+      - <em>Adds migration (run `mix thesis.install && mix ecto.migrate`)</em>.
+      <br/><br/>
+      <strong>pre 0.2.0</strong><br/>
+      - Core Thesis functionality: content areas, global content areas, file uploads, import/export, dynamic pages, page templates.
+    </td>
+  </tr>
+
```

Comment:
Added a section for changelog:
![20180328_u657r](https://user-images.githubusercontent.com/1775841/38073365-8ef469a8-32df-11e8-8a3a-4a9fbb419ab0.png)

Reply:
Epic